home/.bin/story.rb

```diff
@@ -0,0 +1,52 @@
+require 'shellwords'
+require 'yaml'
+require 'active_support'
+require_relative 'git_repository'
+require_relative 'pivotal_project'
+
+class Story
+  def initialize(story_name:, project_id:, pivotal_token:)
+    @story_name = story_name
+    @project_id = project_id
+    @pivotal_token = pivotal_token
+  end
+
+  private
+
+  attr_reader :story_name, :project_id, :pivotal_token
+
+  def validate_environment_and_arguments
```

Comment:
yeah starting to think more of these methods can have `!`, because they cause exceptions if things are broke